src/components/screens/send/transactionStatus/transactionStatus.js

```diff
@@ -68,11 +71,26 @@ const TransactionStatus = ({
 
   useEffect(() => {
     recipientAccount.loadData({ address: fields.recipient.address });
-    broadcast();
-
     return resetTransactionResult;
   }, []);
 
+  // Broadcast transacion when signedTransaction is updated
```

Comment:
Add a TODO to remove this logic when we implement https://github.com/LiskHQ/lisk-desktop/issues/3801